radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/SimulationTest.java

```diff
@@ -80,6 +81,9 @@
  * High level BFT Simulation Test Runner
  */
 public class SimulationTest {
+	private static final String ENVIRONMENT_VAR_NAME = "TEST_DURATION"; // Same as used by regression test suite
+	private static final Duration DEFAULT_TEST_DURATION = Duration.of(5).seconds();
```

Comment:
This is kind of neat.  Maybe more complicated than strictly necessary, but looks good.